src/interface/web/app/share/chat/layout.tsx

```diff
@@ -20,9 +20,9 @@ export default function RootLayout({
                 httpEquiv="Content-Security-Policy"
                 content="default-src 'self' https://assets.khoj.dev;
                        script-src 'self' https://assets.khoj.dev 'unsafe-inline' 'unsafe-eval';
-                       connect-src 'self' https://ipapi.co/json ws://localhost:42110;
+                       connect-src 'self' data: blob: https://ipapi.co/json ws://localhost:42110;
                        style-src 'self' https://assets.khoj.dev 'unsafe-inline' https://fonts.googleapis.com;
-                       img-src 'self' data: https://*.khoj.dev https://*.googleusercontent.com https://*.google.com/ https://*.gstatic.com;
+                       img-src 'self' data: blob: https://*.khoj.dev https://*.googleusercontent.com https://*.google.com/ https://*.gstatic.com;
```

Comment:
Why is blob:` required for img-src in CSP? For `img-src` is `data:` not good enough to display base64 encoded image being uploaded?

Note: similar changes are done to the CSP in `src/interface/web/app/layout.tsx` and `src/interface/web/app/chat/layout.tsx`

Reply:
uploaded images come in as blob so this needs to be added to render from blob!